how2meet/db/models.py

```diff
@@ -40,9 +39,9 @@ class Guest(Base):
     id = Column(String(36), primary_key=True, index=True)
     name = Column(String(100))
     email = Column(String(100), nullable=True)
-    phone = Column(Integer(), nullable=True)
-    status = Column(String(15))
-    event_id = Column(String(36), ForeignKey("events.id"), nullable=False)
+    phone = Column(String(15), nullable=True)
```

Comment:
We should use the Pydantic extra types module for phone numbers here. The work has already been done for us :) 

https://docs.pydantic.dev/latest/api/pydantic_extra_types_phone_numbers/

Reply:
Would you believe I'd already written a [ticket](https://www.notion.so/connor-rubin/Fix-types-224a17199e1d42c9b10b750bae109b0b?pvs=4) for it a couple weeks ago?  lol